library/core/src/num/mod.rs

```diff
@@ -65,6 +66,13 @@ pub use error::TryFromIntError;
 )]
 pub use error::IntErrorKind;
 
+#[unstable(
+    feature = "sealed_int_traits",
+    reason = "can be used to write generic code over primitive integers",
```

Comment:
"reason" here is the reason for the unstable marker, not the reason they exist. This should be something like "newly added".

Reply:
Or should I just leave reason out? I tried to find an existing reason message for similarly unstable things, but didn't find any; the most common case is to have no reason field.

Reply:
I have removed the reason field for the sealed_int_traits feature.